src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  onTabSelected,
+  selectedTabId,
+}) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            className={cn('', {
+              'is-active': tab.id === selectedTabId,
```

Comment:
```suggestion
              'is-active': tab.id === selectedTab.id,
```

same as below, create a separate variable with a default value